src/parsec_client/operations/service_api_coverage.md

```diff
@@ -58,6 +62,23 @@ for key management operations.
 | DhKeyPair    | ✅                  | ❌              | ❌              |
 | DhPublicKey  | ✅                  | ❌              | ❌              |
 
+### Elliptic curve families
```

Comment:
Nice 👌 !

Reply:
Indeed, I don't know why we forgot this when we made the page